src/blueprints/tv2/action-factories/tv2-split-screen-action-factory.ts

```diff
@@ -229,13 +246,12 @@ export class Tv2SplitScreenActionFactory {
     for (let inputIndex = 0; inputIndex < NUMBER_OF_SPLIT_SCREEN_BOXES; inputIndex++) {
       const actionsForInput: Tv2SplitScreenInsertSourceInputAction[] = sources
         .map(source => {
-
-          const audioTimelineObjects: TimelineObject[] = this.audioTimelineObjectFactory.createTimelineObjectsForSource(blueprintConfiguration, source, audioMode)
+          const audioTimelineObjects: Tv2BlueprintTimelineObject[] = this.audioTimelineObjectFactory.createTimelineObjectsForSource(blueprintConfiguration, source, audioMode)
 
           return {
-            id: `insert_${name}_${source.SourceName}_to_split_screen_input_${inputIndex}_action`,
-            name: `Insert ${name} ${source.SourceName} in DVE input ${inputIndex}`,
-            description: `Insert ${name} ${source.SourceName} in DVE input ${inputIndex}`,
+            id: `insert_${name}_${source.name}_to_split_screen_input_${inputIndex}_action`,
```

Comment:
It looks like `source.name` need to be sanitized for spaces.

Reply:
Updated to replace all whitespaces with underscores in the id.